plugins/inputs/win_services/win_services.go

```diff
@@ -94,6 +95,7 @@ type WinServices struct {
 	Log telegraf.Logger
 
 	ServiceNames []string `toml:"service_names"`
+	IgnoredNames []string `toml:"ignored_names"`
```

Comment:
Would you be ok with renaming this to 'ServiceNamesExcluded' to make it more obvious what the config does?

Reply:
o.k. no problem